calblueprint/ogsc
# Combobox for linked players + beginning steps for adding viewing permission relations

# Your feature title here

- Created Combobox for adding linkedPlayers on the admin user creation page (thanks to @ethanlee16 's suggestion of using [Downshift Combobox](https://www.downshift-js.com/use-combobox))
- Began adding framework in API to link users with viewing permissions

## TODO (if applicable)
 (next sprint)
- [ ] Create player cards with delete functionality to display above the combobox
- [ ] Finish endpoint tasks to include the update to `viewedByPermissions`and `viewerPermissions`

## How to Test/Use PR feature

- Go to http://localhost:3000/admin/invite/new
- Select mentor, parent, or donor as role in order to see the combobox

## PR Checklist

Does your branch (check if true):

## Screenshots
![image](https://user-images.githubusercontent.com/34170094/99056561-5003c300-254f-11eb-8954-279796e42d9d.png)

<img width="1428" alt="Screen Shot 2020-11-13 at 1 27 14 AM" src="https://user-images.githubusercontent.com/34170094/99056596-601ba280-254f-11eb-8c33-68938ec229f3.png">

![image](https://user-images.githubusercontent.com/34170094/99056667-7f1a3480-254f-11eb-8bec-3dc88bf573b8.png)

`selected` to be updated with player cards (see TODO)
![image](https://user-images.githubusercontent.com/34170094/99056799-b0930000-254f-11eb-96fa-46aa154e1289.png)

CC: @erinysong
